src/app/map/[id]/components/PlacedMarkers.tsx

```diff
@@ -12,12 +12,24 @@ import type { FeatureCollection, Point } from "geojson";
 
 export default function PlacedMarkers() {
   const { viewConfig } = useContext(MapContext);
-  const { placedMarkers, selectedPlacedMarkerId } =
+  const { folders, placedMarkers, selectedPlacedMarkerId } =
     useContext(MarkerAndTurfContext);
 
+  const visiblePlacedMarkers = useMemo(() => {
+    return placedMarkers.filter((marker) => {
+      if (!marker.folderId) return true;
+
+      const parentFolder = folders.find(
+        (folder) => folder.id === marker.folderId,
+      );
+
+      return !parentFolder?.hideMarkers;
+    });
+  }, [placedMarkers, folders]);
```

Comment:
Not sure if that's the best place to do that but it seemed the most straight-forward to me